api/models/session.py

```diff
@@ -0,0 +1,9 @@
+from sqlalchemy import Column, Integer, String, DateTime, func
+from sqlalchemy_utils import UUIDType
+from models import db
+
+
+class Session(db.Model):
```

Comment:
If each user can only have 1 session, call this `CurrentSession` or something

Reply:
Fixed in 3efe221